pkgs/development/python-modules/playwright/driver.nix

```
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchzip
```

```suggestion
, fetchurl
```

(see my other comment)